settings.py

```diff
@@ -1375,7 +1374,6 @@
     'formplayer',
     'phonelog',
     'registration',
-    'wisepill',
```

Comment:
@dannyroberts This is in `COUCHDB_APPS`. How important is it to be diligent about removing things from here as couch views are deleted?

Reply:
I'm not sure. Presumably if we reference an app that doesn't exist it would throw an error? If that's not true, there may not be any other repercussions of not removing it, so then it would just be good cleanup which is good too